app/v2/schema.js

```diff
@@ -0,0 +1,8 @@
+const Joi = require('joi')
+
+module.exports = Joi.object({
+  type: Joi.string().required(),
+  source: Joi.string().required(),
+  subject: Joi.string().optional(),
+  data: Joi.any().optional()
+}).required()
```

Comment:
Should the name of this file be 'event-schema' or do you think 'schema' is clear enough as it is?

Reply:
I thought `schema` was clear enough as it's the only thing we're validating in the repo.  Happy to change it though if you think it's clearer to include `event` as a prefix?

Reply:
Yeah, I think so. Keeps it sort of consistent with event-publisher and validate-events

Reply:
Fair point.  Updated :)